tests/unittests/OperatorTest.cpp

```diff
@@ -1358,6 +1358,52 @@ TEST_P(Operator, ChannelShuffle) {
     EXPECT_FLOAT_EQ(results.at({0, i, 0, 0}), expected[i]);
 }
 
+TEST_P(Operator, Squeeze) {
+  auto *inputs = mod_.createVariable(ElemKind::FloatTy, {1, 2, 1, 5}, "inputs");
+  inputs->getHandle() = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+
+  // Test 1:
+  {
+    std::vector<size_t> axes = {0};
+    Node *SQZ = F_->createSqueeze("SQZ", inputs, axes);
+    SaveNode *S = F_->createSave("save", SQZ);
+
+    EE_.compile(CompilationMode::Infer, F_);
+    EE_.run({}, {});
+
+    auto results =
+        llvm::cast<Variable>(S->getOutput())->getPayload().getHandle();
+    std::vector<size_t> expectedDims = {2, 1, 5};
+    EXPECT_TRUE(results.dims().vec() == expectedDims);
+    std::vector<int> expectedValues = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```

Comment:
nit: Perhaps you can move the vector to the top and use it to initialize `inputs` as well, so you don't need to list out `{1, 2, ...}` three separate times.